src/internal/communication.ts

```diff
@@ -312,6 +312,7 @@ function createMessageRequest(func: string, args: any[]): MessageRequest {
   return {
     id: CommunicationPrivate.nextMessageId++,
     func: func,
+    creationTimestamp: Date.now(),
```

Comment:
Why not just call this `timestamp`? The `creationTimestamp` seems a little wordy. @sachingupta